lib/pipeline.rb

```diff
@@ -67,6 +71,10 @@ def documents
     Pipeline::Documents.new(pipeline: self)
   end
 
+  def searches
+    Pipeline::Search.new(pipeline: self)
```

Comment:
Should `Pipeline::Search` be plural?

Reply:
I failed to add specs for the non-admin endpoints!!

Reply:
Fixed and added index spec. Not sure create/delete/update is worth it for this endpoint right now...

Reply:
Sounds great!